README.md

```diff
@@ -43,22 +44,22 @@ The component contain this props:
 
 Should be called this way.
 
-````
+```js
 <SearchBar handleSubmit={this.onSubmit}> 
-  <label className="Label-search">
+  <label className={styles.labelSearch}>
```

Comment:
this needs an html-for so when clicking the label, we also focus the input

https://www.w3schools.com/tags/tryit.asp?filename=tryhtml_label